btk/draw_blends.py

```diff
@@ -467,7 +473,7 @@ def render_single(self, entry, filt, psf, survey, extra_data):
         # around them, so a gaussian kernel convolution smoothes it out.
         # It has the slight disadvantage of adding some band-limitedness to the image,
         # but with a small kernel, it's better than doing nothing.
-        gal = galsim.Convolve(gal, galsim.Gaussian(sigma=2 * survey.pixel_scale))
+        # gal = galsim.Convolve(gal, galsim.Gaussian(sigma=2 * survey.pixel_scale))
```

Comment:
Can we remove this commented out line? (and the comment from Remy above?) Can you remind me why we needed this?

Reply:
Oh yes I left it in case it would be useful for one who wants to simulate images with sharp PSFs (this was the first intention of Remy when he added this). But maybe we can just remember it and remove it from the code.

Reply:
Thanks for the quick reply! Is there a good definition for 'sharp' ? Ideally we can check if the PSF provided by the user is sharp and then apply this in that case. But perhaps it's best if we remove this for now, open a separate issue and resolve this later.

Reply:
I think Remy considered sharp in PSF size in the order of the HST PSF. In any case I think it is sure that this smoothing is not necessary for ground-based instrument simulations. Maybe we could discuss this with Remy indeed.

Reply:
Sounds good just opened #181